LDAR_Sim/inputs_template/P_aircraft.txt

```diff
@@ -1,70 +1,79 @@
 P_aircraft = {
-        'methods': {
-	                 'aircraft': {
-                            'name': 'aircraft',
-                            'n_crews': 1,
-                            'min_temp': -30,
-                            'max_wind': 10,
-                            'max_precip': 1,
-                            'max_workday': 8,
-			    'measurement_scale': "equipment",
-                            'cost_per_day': 10000,
-                            'follow_up_thresh': [0, "absolute"],
-                            'follow_up_prop': 1,
-                            't_lost_per_site': 10,                             
-                            'reporting_delay': 2,
-                            'MDL': 0.01,         
-			    'QE': 0,
-			    'consider_daylight': False                      
-                            },
-                   'OGI_FU': {
-                            'name': 'OGI_FU',
-                            'n_crews': 1,
-            		    'min_temp': -20,
-              		    'max_wind': 10,
-                	    'max_precip': 0.1,
-                            'max_workday': 8,
-                            'cost_per_day': 2500,
-                            'reporting_delay': 2,
-                            'MDL': [0.01275, 2.78e-6],
-			    'consider_daylight': False  
-                            }
+    'methods': {
+        'aircraft': {
+            'label': 'aircraft',
+            'module': 'aircraft',
```

Comment:
Methods should have a 'label', or 'name', or something to refer to them. This could be 'aircraft', but doesn't need to be. It could be 'kairos aircraft' or whatever.

Methods should have a 'module' or some key to know what code to call. This needs to be one of 'aircraft', 'OGI', etc. I think that is the best way to do it.

The 'dict key name' for methods cannot be the 'module', otherwise you cannot run a program with multiple OGIs, etc. because this is a dictionary and you cannot have more than one thing with same dictionary key. It could be the 'name', or 'label'. That's probably OK. Or alternatively change this to list, just like programs one hierarchy up - probably not bad idea really to keep consistency.

My input PR has the exact same solution drafted out, but with different words - I'll adjust to these words - but I fully support this concept.

Program definitions in the default input parameters are kind of 4 things at once: (i) example program with a given method, (ii) a test case with a given method, (iii) a definition of the parameter structure of a given method, and (iv) a definition of the defaults of a given method. The method definitions are delivered presently inside of an example program - not a major issue, but the reason there is an aircraft program I think. Though no real reason this couldn't be mixed up.

Reply:
Moved into an issue #81